src/plots/cartesian/layout_attributes.js

```diff
@@ -698,6 +698,26 @@ module.exports = {
             'In other cases the default is *hide past div*.'
         ].join(' ')
     },
+    ticklabelshift: {
+        valType: 'integer',
+        dflt: 0,
+        editType: 'ticks',
+        description: [
+            'Shifts the tick labels by the specified number of pixels in parallel to the axis.',
+            'Positive values move the labels in the positive direction of the axis.'
+        ].join(' ')
+    },
+    ticklabelstandoff: {
+        valType: 'integer',
+        dflt: 0,
+        editType: 'ticks',
+        description: [
+            'Shifts the tick labels by the specified number of pixels orthogonally to the axis.',
+            'A positive `ticklabelstandoff` will move the labels farther away from the plot area',
+            'if `ticklabelposition` is "outside", and deeper into the plotarea if',
+            '`ticklabelposition` is "inside".'
```

Comment:
```suggestion
            'Sets the standoff distance (in px) between the axis tick labels and their default position.',
            'A positive `ticklabelstandoff` moves the labels farther away from the plot area',
            'if `ticklabelposition` is *outside*, and deeper into the plot area if',
            '`ticklabelposition` is *inside*.'
```

Reply:
Done, also added note about negative values possibly moving inside ticks to the outside and vice versa. I forgot to push that before.